merlin/util/duplicatebcremoval.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from scipy.spatial import cKDTree
+import networkx as nx
+import pandas as pd
+from random import choice
+
+
+def cleanup_across_z(barcodes: pd.DataFrame, zPlanes: int, maxDist: float) \
+        -> pd.DataFrame:
```

Comment:
Can we assert that all the barcodes have the same barcode_id and rename the function to remove_zplane_duplicates_single_barcodeid to make sure this function isn't accidentally misused?